bucket/kubectxwin.json

```diff
@@ -0,0 +1,27 @@
+{
+    "homepage": "https://github.com/thomasliddledba/kubectxwin",
+    "description": "Windows Version of the powerful tool kubectx.",
+    "license": "MIT",
+    "version": "0.1.1",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/thomasliddledba/kubectxwin/releases/download/0.1.1/kubectxwin.exe",
+            "hash": "sha1:34703BCB9B16865541E58524B596B1AEC0E4E3D0"
```

Comment:
Sha1 hashes are provided somewhere? Or you just generated it using sha1?

Reply:
I just generated 😂